explorer-ui/src/components/blocks/BlockList.tsx

```diff
@@ -0,0 +1,153 @@
+import React from "react"
+import { Edge, Page } from "../../types/pagination"
+import BlockRow from "./BlockRow"
+import List, { ListProps } from "../commons/List"
+import Pagination from "../navigation/Pagination"
+import SortBy from "../query/SortBy"
+import ListQuery, { UpdateMode } from "../query/ListQuery"
+import Filters from "../query/Filters"
+import { LightBlock, Block } from "../../types/blocks"
+
+export function buildArrayOf (n: number, f: (index: number) => Object) {
+  return [...Array(n)].map((_, i) => f(i))
+}
+
+export function mockBlock (i: number) {
```

Comment:
Are we still use mock data or we can remove  this?

Reply:
good catch! i'll move them to the tests.